core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2406,4 +2406,15 @@ private CarbonCommonConstants() {
   public static final String BUCKET_COLUMNS = "bucket_columns";
   public static final String BUCKET_NUMBER = "bucket_number";
 
+  /**
+   * property for table level cache expiration
```

Comment:
```suggestion
   * table property name for table level cache expiration. Carbon maintains index cache in driver side and the cache will be expired after seconds indicated by this table property
```

Reply:
I think more description need to add to describe the expiration by time and by maximum size

Reply:
added